app/scripts/directives/moi-sound/moi-sound.js

```diff
@@ -95,6 +96,8 @@
         // TODO support loops
         $cordovaNativeAudio.play(vmSound.sound, replaySound);
       } else {
+        console.log('Volume', vmSound.volume);
+        vmSound.$audio[0].volume = vmSound.volume || 1;
```

Comment:
explota el parlante jajajaja  Specifies the current volume of the audio/video. Must be a number between 0.0 and 1.0.